src/ccall.cpp

```diff
@@ -1690,6 +1691,18 @@ static jl_cgval_t emit_ccall(jl_codectx_t &ctx, jl_value_t **args, size_t nargs)
         ai.decorateInst(tid);
         return mark_or_box_ccall_result(ctx, tid, retboxed, rt, unionall, static_rt);
     }
+    else if (is_libjulia_func(jl_get_tls_world_age)) {
+        ++CCALL_STAT(jl_get_tls_world_age);
+        assert(lrt == ctx.types().T_size);
+        assert(!isVa && !llvmcall && nccallargs == 0);
+        JL_GC_POP();
+        LoadInst *world_age = ctx.builder.CreateAlignedLoad(ctx.types().T_size, ctx.world_age_field, Align(sizeof(ctx.types().sizeof_ptr)));
+        setName(ctx.emission_context, world_age, "task_world_age");
+        jl_aliasinfo_t ai = jl_aliasinfo_t::fromTBAA(ctx, ctx.tbaa().tbaa_gcframe);
+        ai.decorateInst(world_age);
+        world_age->setMetadata(LLVMContext::MD_invariant_group, llvm::MDNode::get(ctx.builder.getContext(), {}));
```

Comment:
I think this needs to check if we're at toplevel, right?

Reply:
Why would it?

Reply:
Because of the stores to the world age in toplevel code ("Step 12")